AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -124,35 +128,42 @@ def AtomEditorComponents_Light_AddedToEntity():
         general.idle_wait_frames(1)
         Report.result(Tests.creation_redo, light_entity.exists())
 
-        # 5. Enter/Exit game mode.
+        # 5. Set the light type to sphere
```

Comment:
Same note as above. We can either try cycling through all light types here, or at least name this generically so that we can cycle through the light types in the future.

Reply:
Updated to cycle through light types, and report result